tools/syncer.md

```diff
@@ -380,11 +391,11 @@ target-table = "order_2017"
         1 row in set (0.00 sec)
```

Comment:
I find some SQL execution result table is not line alignment, we'd can paste execution result directly and it will look better